src/components/organisms/FileTreePane/FileTreePane.tsx

```diff
@@ -425,6 +414,17 @@ const TreeItem: React.FC<TreeItemProps> = props => {
         onClick={e => {
           e.domEvent.stopPropagation();
 
+          onCreateResource(isFolder ? {targetFolder: target} : {targetFile: target});
+        }}
+        key="create_resource"
+      >
+        New Resource
```

Comment:
Can we call it "Add Resource" for files and "New Resource" for folders?

Reply:
yes